packages/components/src/config/colors.ts

```diff
@@ -7,11 +7,12 @@ export const THEME = {
     light: {
         THEME: 'light',
 
-        BG_GREEN: '#00854D',
+        BG_GREEN: '#0F6148',
```

Comment:
I wouldn't tough this file at all for now. Also, better start using colors from the `theme` package, the one used in Figma, check other updated components

Reply:
755f9fbbae98c6194cb5d379ff90e4ea918ce5f3